lucille-core/src/main/java/com/kmwllc/lucille/core/Indexer.java

```diff
@@ -249,17 +258,42 @@ private void sendToIndexWithAccounting(List<Document> batchedDocs) {
           }
         }
       }
+
+      // We've sent a message for each Document. Don't want to run the code below.
```

Comment:
Small note but it took me a while to understand the intention here because I though "the code below" in this comment was referring to the finally block, which didn't make sense functionally or semantically. I understand the intention now, but it would probably be easier to understand by just adding "We don't want to run the code below the finally block".